src/libraries/System.Text.RegularExpressions/src/Resources/Strings.resx

```diff
@@ -225,4 +225,31 @@
   <data name="UnterminatedComment" xml:space="preserve">
     <value>Unterminated (?#...) comment.</value>
   </data>
+  <data name="NotSupported_NonBacktrackingAndReplacementsWithSubstitutions" xml:space="preserve">
+    <value>Regex replacements with substitutions are not supported with RegexOptions.NonBacktracking.</value>
+  </data>
+  <data name="NotSupported_NonBacktrackingConflictingOption" xml:space="preserve">
+    <value>RegexOptions.NonBacktracking is not supported in conjunction with RegexOptions.{0}.</value>
+  </data>
+  <data name="NotSupported_NonBacktrackingConflictingExpression" xml:space="preserve">
+    <value>RegexOptions.NonBacktracking is not supported in conjunction with expressions containing {0}.</value>
```

Comment:
as an aside, this is concatenating sentence fragments, which is not localizable. they either need to follow a period, or be in quotes.

Reply:
We can add single quotes.  It's not meant to be sentence fragments, but rather a description of the thing not supported.  It could be `: '{0}'` if that's better.

Reply:
that would work fine.